demo/api-deployment.yaml

```diff
@@ -14,14 +14,14 @@ spec:
     spec:
       containers:
       - name: spring-boot-app-api
-        image: <private docker repo>/api:v2.1
-        # resources:
-        #   requests:
-        #     memory: "64Mi"
-        #     cpu: "250m"
-        #   limits:
-        #     memory: "128Mi"
-        #     cpu: "500m"
+        image: <private docker repo>/api:v2.0
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "250m"
+          limits:
+            memory: "256Mi"
```

Comment:
Is this a realistic amount of memory for a JVM in prod?

Reply:
Not for a large service, but as far as I can tell, Jackson is pretty small, it's just loading and passing json from the DB. Potentially we could run into memory issues if someone requests for a very large number of items.